packages/preview/starter-journal-article/0.1.1/template/main.typ

```diff
@@ -0,0 +1,38 @@
+#import "@preview/starter-journal-article:0.1.0": article, author-meta
```

Comment:
This says 0.1.0 but the package version is 0.1.1 - that means this import fails

Reply:
Thanks. This error has been fixed.